lib/generated_definitions/fed_ex.rb

```diff
@@ -31,7 +31,7 @@ def self.holidays_by_month
   end
 
 def self.day_after_thanksgiving(year)
-  Holidays::DateCalculatorFactory.day_of_month_calculator.call(year, 11, 4, 4) + 1
+  Date.calculate_mday(year, 11, 4, 4) + 1
```

Comment:
This is a recent change (last two weeks or so) but we do need the fully qualified namespacing here. Can you remove this change? Same with the other cases of this.